### ISSUE TITLE
Adding footer to learning page

### ISSUE THREAD

**Issue by komalioruganti** [fykaa/Sample-Portfolio-for-Learners#67]
There is no footer for the learnings page. Can u assign me this issue?

**Comment by komalioruganti** [comment]
Hey @fykaa . I would like to work on this issue. Can you assign me this?

**Comment by komalioruganti** [comment]
Hey! I made these changes. Can I make a PR now?

**BEFORE:**

![image](https://user-images.githubusercontent.com/95839946/196171354-1c94f86f-066d-4a23-bd59-05f0696b7397.png)

**AFTER:**

![image](https://user-images.githubusercontent.com/95839946/196171063-5a0823e1-474a-4794-b712-e3c4238f10f4.png)

**Comment by unovajnv** [comment]
hey can you  assign it to me , i will make it more beutifull

**Comment by fykaa** [comment]
Sure @unovajnv

**Comment by komalioruganti** [comment]
What do you think about this?
![image](https://user-images.githubusercontent.com/95839946/197147424-3ccf7d52-7d7f-4d24-bbbf-47604e733562.png)

**Comment by fykaa** [comment]
Hi, @komalioruganti Your PR has merge conflicts, Can you please look into it. Thanks :) #79

**Comment by komalioruganti** [comment]
Yeah sure. I actually made a new PR. Can you review it and let me know if there are any more conflicts?  I will resolve them.
